guides/v2.3/graphql/send-request.md

```diff
@@ -3,7 +3,25 @@ group: graphql
 title: GraphQL requests
 ---
 
-In Magento 2.3.1, you must send all queries and mutations as HTTP POST requests. 
+Magento GraphQL supports the HTTP GET and POST methods. You can send a query as a GET or POST request. GET requests can be cached, although Magento does not currently support this feature. Mutations must be POST requests. 
+
+You can optionally send a GET query request in a URL. In these requests, you must specify `query` as the query string. You might need to encode the query.  
+
+`http://<magento_host>/graphql?query=%7Bproducts(filter%3A%7Bsku%3A%7Blike%3A%2224-WB%25%22%7D%7D)%7Bitems%7Bsku%7D%7D%7D`
```

Comment:
```suggestion
`http://<host>/graphql?query=%7Bproducts(filter%3A%7Bsku%3A%7Blike%3A%2224-WB%25%22%7D%7D)%7Bitems%7Bsku%7D%7D%7D`
```
our rest documentation just uses `host` at the beginning of the request URL. Do GraphQL URL requests need to be different?